css/src/components/badge.scss

```diff
@@ -8,7 +8,12 @@ $badge-font-size-sm: $font-size-9 !default;
 $badge-font-size-lg: $font-size-7 !default;
```

Comment:
At the top of this file, perhaps we should adjust padding-block to be .5em as well. This still looks really, really close to the Figmas and also allows for easy icon-only badge padding.